src/javascripts/helpers/data/authData.js

```diff
@@ -0,0 +1,18 @@
+
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
```

Comment:
can we add all the jquery selectors up here - that way if a class or id changes we only have to do it once
const logoutButton = $('#navbar-logout-button');

Reply:
breaks code if I give it a const variable name and then that variable to add/remove